recipes/master.rb

```diff
@@ -116,3 +115,18 @@
   owner node['jenkins']['master']['user']
   group node['jenkins']['master']['group']
 end
+
+node.deep_fetch('osl-jenkins', 'gems').each do |g|
```

Comment:
What is the purpose of `deep_fetch`?

Reply:
If the node hash doesn't exist, don't error out.